operations/enable-nfs-volume-service.yml

```diff
@@ -93,7 +94,7 @@
     instances: 1
     azs: [z1]
     lifecycle: errand
-    vm_type: minimal
+    vm_type: default
```

Comment:
We should probably keep the `vm_type: minimal`. It doesn't really make sense to call a vm_type "default"